src/overlays/actors/ovl_En_Nb/z_en_nb.h

```diff
@@ -2,18 +2,69 @@
 #define Z_EN_NB_H
 
 #include "global.h"
+#include "objects/object_nb/object_nb.h"
 
 struct EnNb;
 
 typedef void (*EnNbActionFunc)(struct EnNb*, PlayState*);
+typedef s32 (*EnNbUnkFunc)(Actor*, PlayState*);
+
+typedef enum EnNbAnimation {
+    /* -1 */ EN_NB_ANIM_INVALID = -1,
+    /*  0 */ EN_NB_ANIM_0,
+    /*  1 */ EN_NB_ANIM_1,
+    /*  2 */ EN_NB_ANIM_TALK_ONCE,
+    /*  3 */ EN_NB_ANIM_TALK_LOOP,
+    /*  4 */ EN_NB_ANIM_ANGRY,
+    /*  5 */ EN_NB_ANIM_RELIEVED,
+} EnNbAnimation;
```

Comment:
Probably better in the c file

Reply:
Marked as resolved, but no changes or comment?

Reply:
Whoops, my bad.
I left the enum in the header since I'm using the `EnNbAnimation` as the type of the `animIndex` member. I guess I could move the enum to the C file and just use a plain `s32`, but I kinda prefer using the enum type

Reply:
Ahh, I didn't notice that. It should be fine here than.